xmtp_mls/src/configuration.rs

```diff
@@ -29,7 +29,9 @@ const SYNC_UPDATE_INSTALLATIONS_INTERVAL_NS: i64 = NS_IN_HOUR / 2; // 30 min
 
 pub const SEND_MESSAGE_UPDATE_INSTALLATIONS_INTERVAL_NS: i64 = 5 * NS_IN_SEC;
 
-pub const MAX_GROUP_SIZE: usize = 220;
+pub const MAX_GROUP_SIZE: usize = 250;
```

Comment:
Bump this up to 250 so it's a more square number and matches other protocols limits.